### ISSUE TITLE
BatchNormalization inference equation in doc is incorrect

### ISSUE THREAD

**Issue by brian36** [tensorflow/tensorflow#46522]
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

## Description of issue (what needs changing):

This documentation states the following:

> During inference (i.e. when using evaluate() or predict() or when calling the layer/model with the argument training=False (which is the default), the layer normalizes its output using a moving average of the mean and standard deviation of the batches it has seen during training. That is to say, it returns (batch - self.moving_mean) / (self.moving_var + epsilon) * gamma + beta.

This equation is incorrect, testing with the source code shows it gives the wrong output.  The correct equation that gives the right output (and matches the correct equation from literature) is:

`(batch - self.moving_mean) / sqrt(self.moving_var + epsilon) * gamma + beta.`

I.e., it is missing square-root.

Further, for more clarity (to avoid confusion for some) it would be better to write it as:

`gamma*(batch - self.moving_mean) / sqrt(self.moving_var + epsilon) + beta.`
